episodes/26-external.md

```diff
@@ -102,10 +102,68 @@ You can encourage contributions to your lesson from newcomers by using specific
+GitHub implements a comprehensive [notifications system](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) to keep you up-to-date with activities in the lesson repository. 
 
 Check out [GitHub's documentation on setting notifications on individual repository](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository). You can choose whether to watch or unwatch an individual repository, or can choose to only be notified of certain event types such as issues, pull requests, mentions, etc.
 
+:::::::::::::::::::::::::::::::::::::::::: callout
+
+## Managing GitHub Notifications
+
+If you work on multiple projects,
+or the projects you follow on GitHub are particularly active,
+the volume of notifications you receive can quickly become overwhelming.
+Here are some approaches you can take to help you stay on top of things, 
+and distinguish the high-priority tasks and important updates
+from the regular traffic.
+
+### Email Notifications
+
+If you want to filter, organise, and redirect email notifications from GitHub,
+here are some characteristics of the messages that you make use of:
+
+- All GitHub notifications are sent from the address `notifications@github.com`.
+- Notification emails are sent to the address `repo-name@noreply.github.com`, 
+  where `repo-name` is the name of the repository where the notification was triggered.
+- The email subject begins with `[org-or-user/repo-name]`,
+  where `org-or-user` is the name of the organisation or the username of the user who owns the repository
+  and `repo-name` is the name of the repository where the notification was triggered.
+- In addition to your email address, 
+  the cc field of the message contains an address that describes
+  the type of event that triggered the notification,
+  e.g. `author@noreply.github.com` for activity on an issue or pull request that you opened,
+  `mention@noreply.github.com` for a mention of your username, 
+  or `team-mention@noreply.github.com` for a mention of a team you are a member of,
+  etc.
+
+Most email clients provide configuration for rules that can be set to
+redirect messages to particular folders, and/or to annotate them with a mark or flag,
+based on this kind of information. 
+[Here is the documentation for setting such rules in Gmail](https://support.google.com/mail/answer/6579?hl=en).
```

Comment:
GitHub has this documentation about filtering email notifications, which is very helpful: https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#filtering-email-notifications

One that's not mentioned in your list that I had discovered quite by accident last year is that GitHub provides a `mailing-list` header field, which means that in Gmail, you can filter messages for `list(datacarpentry.github.com)` to get everything from datacarpentry and `list(r-socialsci.datacarpentry.github.com)` to get everything from the R for Social Scientists repo. 

Tags in GMail also work like folders so you can assign these tags and nest them within each other.

Reply:
Thanks @zkamvar I have added a point about the `mailing-list` header field in 7990897